utils.go

```diff
@@ -0,0 +1,32 @@
+package firecracker
+
+import (
+	"context"
+	"fmt"
+	"time"
+)
+
+const (
+	defaultAliveVMMCheckDur = 10 * time.Millisecond
+)
+
+// waitForAliveVMM will check for periodically to see if the firecracker VMM is
+// alive. If the VMM takes too long in starting, an error signifying that will
+// be returned.
+func waitForAliveVMM(ctx context.Context, client *Client, maxRetries int) error {
```

Comment:
I'd prefer to see this use `<-ctx.Done` for cancellation instead of a `maxRetries`.  Callers can then use `context.WithDeadline` to achieve a timeout.

Reply:
Done!